### ISSUE TITLE
Starting container with defaults fails

### ISSUE THREAD

**Issue by ernoaapa** [appcelerator/docker-etcd#3]
If I try to start as read in README:
```
docker run -d -p 2379:2379 -p 2380:2380 appcelerator/etcd
```

I get:
```
2017-04-27 11:47:45 I | Running etcd Version: 3.1.5
Git SHA: 20490ca
Go Version: go1.7.5
Go OS/Arch: linux/amd64
setting default urls for peers
setting default urls for client
setting initial advertise peer urls and initial cluster
warning: unable to resolve this container's IP (), revert to /etc/hosts
/bin/run.sh: line 64: exec: -d: invalid option
exec: usage: exec [-cl] [-a name] [command [arguments ...]] [redirection ...]
```

#### Workaround:
Pass all arguments to the container like in README:
`docker run -d -p 2379:2379 -p 2380:2380 appcelerator/etcd --listen-client-urls http://localhost:2379 --advertise-client-urls http://localhost:2379 --listen-peer-urls=http://localhost:2380 --initial-advertise-peer-urls=http://localhost:2380
`

**Comment by ndegory** [comment]
thank you for the report @ernoaapa , we're working on it for version 3.1.6.

**Comment by ndegory** [comment]
fixed in tag 3.1.6, it will work correctly if no option is passed.
